specification/resources/resource-manager/Microsoft.Solutions/stable/2019-03-01/managedapplications.json

```diff
@@ -494,6 +488,65 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Solutions/applicationDefinitions/{applicationDefinitionName}/applicationArtifacts/{applicationArtifactName}": {
+      "get": {
+        "tags": [
+          "ApplicationDefinitionArtifact"
+        ],
+        "operationId": "ApplicationDefinitionArtifact_Get",
+        "description": "Gets the managed application definition artifact.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group. The name is case insensitive.",
+            "pattern": "^[-\\w\\._\\(\\)]+$",
+            "minLength": 1,
+            "maxLength": 90
+          },
+          {
+            "name": "applicationDefinitionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the managed application definition.",
+            "minLength": 3,
+            "maxLength": 64
+          },
+          {
+            "name": "applicationArtifactName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the managed application definition artifact.",
+            "minLength": 3,
+            "maxLength": 64
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
please include the schema for this. Isnt it strongly typed?

Reply:
No, this one returning blob content directly. See our code base in applicationdefinitionartifactcontroller.cs
The returned content will have different schema according to the input applicationArtifactName.

Reply:
@ravbhatnagar , please review the comment @dandanwang0320 replied.